### PR TITLE
Feat/add apeswap bsc

### PR DESCRIPTION
**Pull request by dtmkeng**
```
Sample pools: [
  {
    pool: '0xc087c78abac4a0e900a327444193dbf9ba69058e',
    chain: 'Binance',
    project: 'apeswap-amm',
    symbol: 'USDC-BUSD',
    tvlUsd: 3240900.846657541,
    apy: 1.496213545197944
  },
  {
    pool: '0x2e707261d086687470b515b320478eb1c88d49bb',
    chain: 'Binance',
    project: 'apeswap-amm',
    symbol: 'USDT-BUSD',
    tvlUsd: 3151093.2552148886,
    apy: 1.439865555065283
  },
  {
    pool: '0xf65c1c0478efde3c19b49ecbe7acc57bb6b1d713',
    chain: 'Binance',
    project: 'apeswap-amm',
    symbol: 'BANANA-WBNB',
    tvlUsd: 2933599.7534356527,
    apy: 22.22293043470871
  },
  {
    pool: '0x51e6d27fa57373d8d4c256231241053a70cb1d93',
    chain: 'Binance',
    project: 'apeswap-amm',
    symbol: 'WBNB-BUSD',
    tvlUsd: 2147115.340745578,
    apy: 2.8527360623763114
  },
  {
    pool: '0xa0c3ef24414ed9c9b456740128d8e63d016a9e11',
    chain: 'Binance',
    project: 'apeswap-amm',
    symbol: 'ETH-WBNB',
    tvlUsd: 1926726.5193498817,
    apy: 3.6500164500749936
  },
  {
    pool: '0x7bd46f6da97312ac2dbd1749f82e202764c0b914',
    chain: 'Binance',
    project: 'apeswap-amm',
    symbol: 'BANANA-BUSD',
    tvlUsd: 1752253.5817481517,
    apy: 36.49327214064712
  },
  {
    pool: '0x29a4a3d77c010ce100a45831bf7e798f0f0b325d',
    chain: 'Binance',
    project: 'apeswap-amm',
    symbol: 'WBNB-MATIC',
    tvlUsd: 1333926.0119471157,
    apy: 2.7848561707290247
  },
  {
    pool: '0x8b6ecea3e9bd6290c2150a89af6c69887aaf1870',
    chain: 'Binance',
    project: 'apeswap-amm',
    symbol: 'DAI-BUSD',
    tvlUsd: 1072932.725911569,
    apy: 2.325805515271442
  },
  {
    pool: '0x1e1afe9d9c5f290d8f6996ddb190bd111908a43d',
    chain: 'Binance',
    project: 'apeswap-amm',
    symbol: 'BTCB-WBNB',
    tvlUsd: 945368.326832159,
    apy: 3.239559221933704
  },
  {
    pool: '0x1d0c3044ebf055986c52d38b19b5369374e6bc6a',
    chain: 'Binance',
    project: 'apeswap-amm',
    symbol: 'SUSHI-WBNB',
    tvlUsd: 934745.6925467739,
    apy: 1.1527983336541687
  }
]
```

### CODE REVIEW

**Comment by 0xvrotend** [comment]
Hello, thanks for the PR.
You forgot to calculate the apyBase field (apy from swap fees).
Apeswap has subgraph, so it is easy to add it.

https://bnb.apeswapgraphs.com/subgraphs/name/ape-swap/apeswap-subgraph/graphql

**Comment by dtmkeng** [comment]
done!, 
```
  {
    pool: '0x1D0C3044eBf055986c52D38b19B5369374E6Bc6A',
    chain: 'Binance',
    project: 'apeswap-amm',
    symbol: 'SUSHI-WBNB',
    tvlUsd: 943713.1219724433,
    apyBase: 1.011172053893197,
    apyReward: 1.1591767058128575,
    underlyingTokens: [
      '0x947950BcC74888a40Ffa2593C5798F11Fc9124C4',
      '0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c'
    ],
    rewardTokens: [ '0x603c7f932ED1fc6575303D8Fb018fDCBb0f39a95' ]
  }
```

**Comment by dtmkeng** [comment]
I have a question, how do you find this endpoint?

File: src/adaptors/pancakeswap/index.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const { masterChefABI, lpTokenABI } = require('./abis');
 const utils = require('../utils');
 const { fetchURL } = require('../../helper/utils');
 
-const RPC_URL = 'https://bsc-dataseed1.binance.org/';
+const RPC_URL = 'https://bsc-dataseed1.binance.org/';z
```
Comment (slasher125) [inline src/adaptors/pancakeswap/index.js]: likely typo, pls revert that

Comment (dtmkeng) [reply src/adaptors/pancakeswap/index.js]: oh sry. done!

**Comment by llamatester** [comment]
The apeswap-amm adapter exports pools: 
        
 
 Test Suites: 1 passed, 1 total
    Tests:       689 passed, 689 total
    Snapshots:   0 total
    Time:        0.668 s
    Ran all test suites.
    
    Nb of pools: 98
     
    
    Sample pools:
    ┌─────────┬──────────────────────────────────────────────┬───────────┬───────────────┬───────────────┬────────────────────┬─────────────────────┬────────────────────┬────────────────────────────────────────────────────────────────────────────────────────────────┬──────────────────────────────────────────────────┐
    │ (index) │                     pool                     │   chain   │    project    │    symbol     │       tvlUsd       │       apyBase       │     apyReward      │                                        underlyingTokens                                        │                   rewardTokens                   │
    ├─────────┼──────────────────────────────────────────────┼───────────┼───────────────┼───────────────┼────────────────────┼─────────────────────┼────────────────────┼────────────────────────────────────────────────────────────────────────────────────────────────┼──────────────────────────────────────────────────┤
    │    0    │ '0xC087C78AbaC4A0E900a327444193dBF9BA69058E' │ 'Binance' │ 'apeswap-amm' │  'USDC-BUSD'  │ 3240783.476627921  │ 0.1645709820022277  │ 1.5369941674338587 │ [ '0x8AC76a51cc950d9822D68b83fE1Ad97B32Cd580d', '0xe9e7CEA3DedcA5984780Bafc599bD69ADd087D56' ] │ [ '0x603c7f932ED1fc6575303D8Fb018fDCBb0f39a95' ] │
    │    1    │ '0x2e707261d086687470B515B320478Eb1C88D49bb' │ 'Binance' │ 'apeswap-amm' │  'USDT-BUSD'  │ 3153620.4088276476 │ 0.7755838856967178  │ 1.4778715545751167 │ [ '0x55d398326f99059fF775485246999027B3197955', '0xe9e7CEA3DedcA5984780Bafc599bD69ADd087D56' ] │ [ '0x603c7f932ED1fc6575303D8Fb018fDCBb0f39a95' ] │
    │    2    │ '0xF65C1C0478eFDe3c19b49EcBE7ACc57BB6B1D713' │ 'Binance' │ 'apeswap-amm' │ 'BANANA-WBNB' │ 2989611.9617189434 │          0          │ 22.40011632866557  │ [ '0x603c7f932ED1fc6575303D8Fb018fDCBb0f39a95', '0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c' ] │ [ '0x603c7f932ED1fc6575303D8Fb018fDCBb0f39a95' ] │
    │    3    │ '0x51e6D27FA57373d8d4C256231241053a70Cb1d93' │ 'Binance' │ 'apeswap-amm' │  'WBNB-BUSD'  │ 2163411.210956493  │  3.782869524622994  │ 2.9083106937395535 │ [ '0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c', '0xe9e7CEA3DedcA5984780Bafc599bD69ADd087D56' ] │ [ '0x603c7f932ED1fc6575303D8Fb018fDCBb0f39a95' ] │
    │    4    │ '0xA0C3Ef24414ED9C9B456740128d8E63D016A9e11' │ 'Binance' │ 'apeswap-amm' │  'ETH-WBNB'   │ 1975678.8861827801 │ 0.9099812928566436  │ 3.656465222931608  │ [ '0x2170Ed0880ac9A755fd29B2688956BD959F933F8', '0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c' ] │ [ '0x603c7f932ED1fc6575303D8Fb018fDCBb0f39a95' ] │
    │    5    │ '0x7Bd46f6Da97312AC2DBD1749f82E202764C0B914' │ 'Binance' │ 'apeswap-amm' │ 'BANANA-BUSD' │ 1776279.873444432  │ 0.9251474972975972  │  36.9795205590345  │ [ '0x603c7f932ED1fc6575303D8Fb018fDCBb0f39a95', '0xe9e7CEA3DedcA5984780Bafc599bD69ADd087D56' ] │ [ '0x603c7f932ED1fc6575303D8Fb018fDCBb0f39a95' ] │
    │    6    │ '0x29A4A3D77c010CE100A45831BF7e798f0f0B325D' │ 'Binance' │ 'apeswap-amm' │ 'WBNB-MATIC'  │ 1368208.7386699908 │ 0.8780919640036041  │ 2.788977821592501  │ [ '0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c', '0xCC42724C6683B7E57334c4E856f4c9965ED682bD' ] │ [ '0x603c7f932ED1fc6575303D8Fb018fDCBb0f39a95' ] │
    │    7    │ '0x8b6EcEA3e9bd6290c2150A89AF6c69887AaF1870' │ 'Binance' │ 'apeswap-amm' │  'DAI-BUSD'   │ 1071445.8448345626 │ 0.05243550041583616 │ 2.3924263230209815 │ [ '0x1AF3F329e8BE154074D8769D1FFa4eE058B1DBc3', '0xe9e7CEA3DedcA5984780Bafc599bD69ADd087D56' ] │ [ '0x603c7f932ED1fc6575303D8Fb018fDCBb0f39a95' ] │
    │    8    │ '0x1D0C3044eBf055986c52D38b19B5369374E6Bc6A' │ 'Binance' │ 'apeswap-amm' │ 'SUSHI-WBNB'  │ 973442.8416987571  │ 1.0445207822642948  │ 1.1371015871873371 │ [ '0x947950BcC74888a40Ffa2593C5798F11Fc9124C4', '0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c' ] │ [ '0x603c7f932ED1fc6575303D8Fb018fDCBb0f39a95' ] │
    │    9    │ '0x1E1aFE9D9c5f290d8F6996dDB190bd111908A43D' │ 'Binance' │ 'apeswap-amm' │  'BTCB-WBNB'  │ 972863.4240266602  │ 1.0439501217018656  │ 3.2336871776610163 │ [ '0x7130d2A12B9BCbFAe4f2634d864A1Ee1Ce3Ead9c', '0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c' ] │ [ '0x603c7f932ED1fc6575303D8Fb018fDCBb0f39a95' ] │
    └─────────┴──────────────────────────────────────────────┴───────────┴───────────────┴───────────────┴────────────────────┴─────────────────────┴────────────────────┴────────────────────────────────────────────────────────────────────────────────────────────────┴──────────────────────────────────────────────────┘

**Pull request merged**